peewee.py

```diff
@@ -1277,7 +1277,9 @@ def python_value(self, value):
             if value == 0:
                 return
             elif self.resolution > 1:
-                value, microseconds = divmod(value, self.resolution)
+                ticks_to_microsecond = 1000000 // self.resolution
+                value, ticks = divmod(value, self.resolution)
+                microseconds = ticks * ticks_to_microsecond
```

Comment:
Nevermind, I was using 3 as the resolution and this just happens to work, but the divide by 1000000 is necessary.